src/BlazorAdmin/Pages/CatalogItemPage/Create.razor

```diff
@@ -82,7 +82,7 @@
                                         </div>
                                     </div>
 
-                                    <div class="form-group">
+                                    @*<div class="form-group">
```

Comment:
Why not just delete this, leaving it as commented out feels kind of messy?

Reply:
Removed